ComponentKit/Core/CKComponentAnimation.mm

```diff
@@ -48,7 +48,14 @@ static CKComponentAnimationHooks hooksForCAAnimation(CKComponent *component, CAA
 }
 
 CKComponentAnimation::CKComponentAnimation(CKComponent *component, CAAnimation *animation)
-: hooks(hooksForCAAnimation(component, animation)) {}
+: hooks(hooksForCAAnimation(component, animation))
+{
```

Comment:
don't change indentation for the sake of it, let's stick with the shorter version